src/molecules/TextLockup/text-lockups.module.scss

```diff
@@ -17,8 +13,73 @@ $subheader-color: $neutral-4;
   margin: 0 0 ru(.167);
 }
 
+.secondaryIcon, .smallScreenIcon {
+  position: absolute;
+  width: 100%;
+  transform: translate(-50%, -50%);
+  top: 50%;
+  left: 50%;
+}
```

Comment:
@cisacke No worries -- this works, so I don't think you have to fight it. I'd say just merge this, and then if you want to pair on getting flexbox to work, I'm totally open to that.